develop-docs/sdk/telemetry/profiles/sample-format-v2.mdx

```diff
@@ -0,0 +1,303 @@
+---
+title: Sample Format V2
+sidebar_order: 40
+---
+
+The *Sample Format V2* is designed for use in continuous profiling. Therefore, a profile can exist independently and be sent in the envelope on its own.
+
+## Data Model
+
+### Example
+
+```json
+{
+  "debug_meta": {
+    "images": [
+      {
+        "debug_id": "5819FF25-01CB-3D32-B84F-0634B37D3BBC",
+        "image_addr": "0x00000001023a8000",
+        "type": "macho",
+        "image_size": 16384,
+        "code_file": "/Library/Developer/CoreSimulator/Volumes/iOS_21C62/Library/Developer/CoreSimulator/Profiles/Runtimes/iOS 17.2.simruntime/Contents/Resources/RuntimeRoot/usr/lib/libLogRedirect.dylib"
+      }
+    ]
+  },
+  "profiler_id": "71bba98d90b545c39f2ae73f702d7ef4",
+  "chunk_id": "3e11a5c9831f4e49939c0a81944ea2cb",
+  "client_sdk": {
+    "name": "sentry.cocoa",
+    "version": "8.36.0"
+  },
+  "measurements": { ... },
+  "platform": "cocoa",
+  "release": "io.sentry.sample.iOS-Swift@8.36.0+1",
+  "environment": "simulator",
+  "version": "2",
+  "profile": {
+    "samples": [
+      {
+        "thread_id": "259",
+        "stack_id": 0,
+        "timestamp": 1724777211.5037799
+      }
+    ],
+    "stacks": [
+      [ 0 ]
+    ],
+    "frames": [
+      {
+        "instruction_addr": "0x000000010232d144",
+        "function": "_ZNK5dyld311MachOLoaded17findClosestSymbolEyPPKcPy"
+      }
+    ],
+    "thread_metadata": {
+      "259": {
+        "name": "main"
+      }
+    }
+  }
+}
+```
+
+### Profile metadata
+
+`debug_meta`
+
+: *Object, required on native platforms.* This carry the same payload as <Link
+to="/sdk/event-payloads/sdk/">`debug_meta` interface</Link>. It is required to
+have it for native platforms in order to symbolicate, otherwise you can omit it.
+
+`profiler_id`
+
+: *String, required.* Hexadecimal string representing a uuid4 value. The length
+is exactly 32 characters. Dashes are not allowed. Has to be lowercase. 
+Random UUID for each profiler session.
+
+`chunk_id`
+
+: *String, required.* Hexadecimal string representing a uuid4 value. The length
+is exactly 32 characters. Dashes are not allowed. Has to be lowercase.
+This is a Random UUID identifying a chunk.
+
+`client_sdk`
+
+: *Object, optional.* Contains information about the client *SDK*.
+
+It can have the following attributes:
+- `name`
+- `version`
+
+`measurements`
+
+: *Object, optional.* This field contains various metrics measured during the profile collection.
+
+A measurement is a collection of measurement values and a unit for those values.
+It will only be shown in the UI if it's been integrated. Right now, we only
+support 2 metrics:
+- `frozen_frame_renders`: when a frozen frame was detected.
+- `slow_frame_renders`: when a slow frame was detected.
+
+A measurement value can contain the following attributes:
+- `elapsed_since_start_ns`: number of nanoseconds since the start of the profile
```

Comment:
`V2` expects a unix timestamp with microsecond precision (like for samples), in a field called `timestamp`.

https://github.com/getsentry/relay/blob/master/relay-profiling/src/measurements.rs#L25